code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -152,28 +170,33 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
-/obj/item/pneumatic_cannon/proc/updateTank(obj/item/tank/thetank, removing = 0, mob/living/carbon/human/user)
-	if(removing)
-		if(!src.tank)
-			return
-		to_chat(user, "<span class='notice'>You detach \the [thetank] from \the [src].</span>")
-		src.tank.loc = get_turf(user)
-		user.put_in_hands(tank)
-		src.tank = null
-	if(!removing)
-		if(src.tank)
-			to_chat(user, "<span class='warning'>\The [src] already has a tank.</span>")
-			return
-		if(!user.unEquip(thetank))
-			return
-		to_chat(user, "<span class='notice'>You hook \the [thetank] up to \the [src].</span>")
-		src.tank = thetank
-		thetank.loc = src
-	src.update_icons()
+/obj/item/pneumatic_cannon/proc/remove_tank(mob/living/carbon/human/user)
+	if(!tank)
+		return
+	to_chat(user, "<span class='notice'>You detach \the [tank] from \the [src].</span>")
```

Comment:
All these implicit `\the`s are a little painful, but it does look like it's the case throughout the rest of the file as well.
If you're already tearing up the file anyway, it would be nice if you could do away with them.

Reply:
Will do, mostly forgot about them since we have span defines so I just defaulted to what was in the original